src/components/PremiumBlocker.jsx

```diff
@@ -1,18 +1,25 @@
-import React from 'react'
-import { Icon } from 'semantic-ui-react'
-import '../css/PremiumBlocker.css'
+import React from "react";
+import { Icon, Button } from "semantic-ui-react";
+import "../css/PremiumBlocker.css";
+import { Link, useHistory } from "react-router-dom";
```

Comment:
are you using `useHistory` for anything here?

Reply:
nope, it can be removed!